src/test/java/uk/co/ramp/AppConfigTest.java

```diff
@@ -161,7 +161,9 @@ public void randomDataGeneratorNoArgs() throws ConfigurationException {
         containsString("Additional Seed information not provided, using internal random seed."));
   }
 
+  // TODO: can't see why this is failing... it works in debug for me.
```

Comment:
If you add 
  "seed": 123,
within the testSeedOverride/runSettings.json file, it should fix the test